src/subscribe/subscribe.service.ts

```diff
@@ -1,9 +1,9 @@
-import { Injectable } from '@nestjs/common';
-import { EmailService } from 'src/email/services/email.service';
-import { MailingService } from 'src/email/services/mailing.service';
 import { ICreateSubscription } from './interfaces/create-subscription.interface';
```

Comment:
I think it would be better to rename **subscribe** (verb) module and service to **subscription** module and service. verb is not very convenient looks for this names, because for another modules already be used noun for names.